events_emitter/tasks.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
```

Comment:
File generally needs more DEBUG level logs.

Reply:
In every level I put log for every rule and for every expression, So I think it's more enough logs